lib/fizzy/execute.cpp

```diff
@@ -332,6 +332,19 @@ inline constexpr T popcnt(T value) noexcept
     return static_cast<T>(popcount(value));
 }
 
+template <typename T>
+T signbit(T value) noexcept = delete;
+
+inline bool signbit(float value) noexcept
```

Comment:
We may get a proper `constexpr` `signbit` in C++23: https://open-std.org/JTC1/SC22/WG21/docs/papers/2019/p0533r5.pdf